### PR TITLE
ITNFA-19: Added possibility to redirect to the patient page if the patient already exists

### PR DESCRIPTION
**Pull request by dserkowski**

### CODE REVIEW

**Comment by dserkowski** [comment]
Before this, [the PR in M2Sys](https://github.com/IsantePlus/openmrs-module-m2sys-biometrics/pull/11) should be merged

File: api/src/main/resources/messages_fr.properties [hunk 1]
```diff
@@ -94,6 +94,8 @@ registrationapp.biometrics.m2sys.update.button.label=Mise à jour
 registrationapp.biometrics.m2sys.update.question=Mise à jour d'empreinte
 registrationapp.biometrics.m2sys.label=Empreintes digitales
 registrationapp.biometrics.m2sys.errorEngine=Logiciel client M2SYS ne fonctionne pas.
+registrationapp.biometrics.m2sys.register.alreadyExists.importingQuestion.label=Patient already exists
```
Comment (pgesek) [inline api/src/main/resources/messages_fr.properties]: Can you email Nathaelf for translations?

File: omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/FingerprintM2sysFragmentController.java [hunk 2]
```diff
@@ -37,9 +51,14 @@ public SimpleObject enroll(@SpringBean("messageSourceService") MessageSourceServ
...
         try {
-            BiometricSubject result = biometricEngine.enroll(null);
+            EnrollmentResult result = biometricEngine.enroll(null);
             response.put("success", true);
-            response.put("message",result.getSubjectId());
+            response.put("message", result.getBiometricSubject().getSubjectId());
+            response.put("status", result.getEnrollmentStatus().name());
+            if (result.getEnrollmentStatus() == EnrollmentStatus.ALREADY_REGISTERED) {
+            	Patient patient = findByLocalFpId(result.getBiometricSubject().getSubjectId());
+                response.put("patientUuid", patient.getUuid());
```
Comment (pgesek) [inline omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/field/FingerprintM2sysFragmentController.java]: bad identation

File: omod/src/main/webapp/resources/scripts/field/fingerprintM2sys.js [hunk 3]
```diff
@@ -1,16 +1,54 @@
...
+     emr.setupConfirmationDialog({
+        selector: '#imported-patient-dialog',
+        actions: {
+            confirm: function () {
+                var patientUrl = '/' + OPENMRS_CONTEXT_PATH + '/coreapps/clinicianfacing/patient.page?patientId='
+                        + data['patientUuid'];
+                $(location).attr('href', patientUrl);
+            },
+            cancel: function () {
+                m2SysError('Fingerprints already registered, you cannot register the same fingerprints again!');
```
Comment (pgesek) [inline omod/src/main/webapp/resources/scripts/field/fingerprintM2sys.js]: can we use i18n for this?

**Review by pgesek — CHANGES_REQUESTED** [review]

**Review by pawelcieszko — APPROVED** [review]

**Review by pawelcieszko — APPROVED** [review]

**Pull request merged**